REUSE.toml

```diff
@@ -0,0 +1,46 @@
+# ~~~
+# Copyright (c) 2024 The Khronos Group Inc.
+# Copyright (c) 2024 RasterGrid Kft.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ~~~
+version = 1
+SPDX-PackageName = "Vulkan SC Emulation"
+SPDX-PackageSupplier = "The Khronos Group, Inc."
+SPDX-PackageDownloadLocation = "https://github.com/KhronosGroup/VulkanSC-Emulation"
+
+[[annotations]]
+path = [
+    "tests/pcc/data/**",
+    ".clang-format",
+    "icd/generated/.clang-format",
+    "icd/icd_shadow_stack.cpp",
```

Comment:
This can be removed from the list.

Reply:
Removed most. See commit messages and comments.